test/models/waterbody_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class WaterbodyTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Here you could assert the sensors nullify constraint (or delete the file).

Reply:
Fixed.